Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8 AS xaitk_base
+
+RUN pip install poetry
+RUN poetry config virtualenvs.create false
+  
+WORKDIR /xaitk
+
+COPY poetry.lock pyproject.toml /xaitk/
+RUN poetry install --no-root
+  
+COPY . /xaitk/
+RUN poetry install
```

Comment:
I assume that you have these separated so that updates to the XAITK package do not generally necessitate a full dep install when rebuilding. If so, could you comment as such here so that it is made clear?